blocks/header/header.css

```diff
@@ -42,10 +49,364 @@ header nav a:any-link {
   color: currentcolor;
 }
 
+header .nav-menu-overlay {
+  position: absolute;
+  top: 0;
+  z-index: 1;
+  width: 100vw;
+  overflow: hidden;
+  left: 0;
+  min-height: 70rem;
+  display: none;
+}
+
+header .nav-menu-overlay.open {
+  display: block;
+}
+
+header .nav-menu-overlay .nav-menu {
+  background-color: rgb(1 39 64);
+  height: 70rem;
+  width: 100%;
+  background-image: url('../../icons/swoosh/type4/default-m.svg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: top left;
+  background-size: 90%;
+  overflow: visible scroll;
+  padding: 100px 30px;
+  max-width: 1456px;
+  color: #bdccdb;
+}
+
+header .nav-menu-overlay .nav-menu::-webkit-scrollbar{
+  display: none;
+}
+
+header .nav-menu-overlay .nav-menu a{
+ text-decoration: none;
+}
+
+header .nav-menu-overlay .nav-menu a:visited{
+  color: #bdccdb;;
+ }
+
+
+
+
+header .nav-menu-overlay ul li{
+  list-style-type: none;
+}
+
+header .nav-menu-overlay .nav-menu-column.left > ul {
+  padding: 0;
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul {
+  font-family: var(--body-font-family);
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 38.13px;
+  margin: 0;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .nav-menu-overlay-back {
+  position: absolute;
+  left: 30px;
+  padding: 0;
+  margin: 0;
+  z-index: 2;
+  top: 28px;
+  background-color: white;
+  height: 44px;
+  width: 44px;
+  border-radius: 50%;
+  
+}
+
+header .nav-menu-overlay .nav-menu-column.right .highlight-link a{
+color: white;
+font-weight: 800;
+
+
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul li span::before {
+  content: "";
+  width: 7px;
+  height: 2px;
+  top: 15px;
+  border-radius: 20px;
+  position: absolute;
+  left: 15px;
+  background-color: #169af3;
```

Comment:
Use vars for colors. Please check everywhere.

Reply:
Done